recipes/netpbm/meta.yaml

```
@@ -0,0 +1,62 @@
+{% set name = "netpbm" %}
+{% set version = "10.73.28" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://sourceforge.net/projects/netpbm/files/super_stable/10.73.28/netpbm-10.73.28.tgz
+  fn: {{ name }}-{{ version }}.tgz
+  sha256: c0d32d4b9a53fde47dd9a36f45653e0ef51f6c390517f10b0c5056d1a8a844bd
+
+build:
+  number: 0
+  skip: True  # [osx or win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - pkg-config  # [linux]
+    - flex
+    - gettext
+    - make
+  host:
+    - perl
+    - jpeg
+    - libtiff
+    - zlib
+    - libpng
+    - libxml2
+  run:
+    - perl
+    - jpeg
+    - libtiff
+    - zlib
+    - libpng
+    - libxml2
+
+test:
+  commands:
+    - pnmtopng -version
+    - pnmtops -version
+    - ppmtogif -version
+    - pamscale -version
+    - tifftopnm -version
+    - jpegtopnm -version
+    - ppmtopgm -version
+    - pamcut -version
+    - bmptopnm -version
+
+about:
+  home: http://netpbm.sourceforge.net
+  license: GPLv2
```

Is this licenced under `GPL-2.0`? The licence file[1] seems to state it's a complicated mix of licences (they all seem suitable for conda-forge). Maybe it would be better to remove `license_family` and make this:
```suggestion
  license: OTHER
```

[1] http://netpbm.svn.code.sourceforge.net/p/netpbm/code/trunk/doc/copyright_summary